docs/tutorials/model_management.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>We now use <code>Models.select_region()</code> to get a subset of models contributing to a particular region. You can also use <code>Models.select_mask()</code> to get models lying inside the True region of a mask map.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>